### PR TITLE
[APP] #133: update api

### PR DESCRIPTION
**Pull request by furnaxe**

### CODE REVIEW

File: App/lib/Registration/login.dart [hunk 1]
```diff
@@ -193,7 +193,7 @@ class Login_state extends State<Login> {
                 ),
               );
               MaterialPageRoute(builder: (context) => const HomeAdmin());
-            } else if (await fetch_roles() == 'user') {
+            } else if (await fetch_roles() == 'student') {
               // ignore: use_build_context_synchronously
```
Comment (Necrelox) [inline App/lib/Registration/login.dart:197]: why ???????

File: App/lib/api/api.dart [hunk 2]
```diff
@@ -8,7 +8,7 @@ import 'package:flutter/material.dart';
 
 import '../dashboard/Home.dart';
 
-var ip = 'http://10.0.2.2:3002';
```
Comment (Necrelox) [inline App/lib/api/api.dart:11]: WTFFFFF NO ENVIRONNEMENT VARIABLE WTFFF OMG

File: App/lib/api/api.dart [hunk 3]
```diff
@@ -8,7 +8,7 @@ import 'package:flutter/material.dart';
...
+var ip = 'http://51.91.96.23:3000';
 var token = '';
```
Comment (Necrelox) [inline App/lib/api/api.dart:12]: WTF WHY A VAR HERE NOT LOCAL SOTRAGE FOR TOKEN OMG

**Review by Necrelox — CHANGES_REQUESTED** [review]

File: App/lib/Registration/login.dart (hunk 1 above)
Comment (furnaxe) [reply App/lib/Registration/login.dart:197]: tg boomer

**Review by Necrelox — APPROVED** [review]

**Pull request merged**